src-java/testing/functional-tests/src/main/groovy/org/openkilda/functionaltests/helpers/model/FlowStatusHistoryEvent.groovy

```diff
@@ -3,7 +3,8 @@ package org.openkilda.functionaltests.helpers.model
 
 enum FlowStatusHistoryEvent {
     UP("UP"),
-    DELETED("DELETED")
+    DELETED("DELETED"),
+    DEGRADED("DEGRADED")
 
     String status
 
```

Comment:
since the status value and the enum value exactly match, we might also use the following approach :

`enum FlowStatusHistoryEvent {
    UP, DELETED, DEGRADED;

    public static FlowStatusHistoryEvent getByValue(String value) {
        try {
            return FlowStatusHistoryEvent.valueOf(value);
        } catch (IllegalArgumentException e) {
            throw new IllegalArgumentException("Invalid value for flow status in the history flow status response: " + value, e);
        }
    }
}`

Or even more, remove getByValue method and use native `valueOf` method for the enum. (in this case we need to separately handlethe illegalArgumentException and add the appropriate message if needed)